src/components/ListItem.jsx

```diff
@@ -50,6 +55,8 @@ export function ListItem({ name, itemId, dateLastPurchased }) {
 						disabled={check}
```

Comment:
I'm not sure why this is happening, but I noticed that in the app, check marks no longer show up when an item is checked (but `updateItem` still runs, and the urgency changes). The checkboxes also are no longer disabled after checking an item, so it's possible to purchase an item multiple times in a row.

Reply:
Thanks @sarahmjenkins! Definitely something we noticed, and might have to put that on the backlog!

Reply:
@arcan9 I just played around with it a little bit, and I think the issue is what's in my comment above (using Math.ceil in getDaysBetweenDates). Using the previous code (what was in useEffect) seemed to fix it:

`		const currentDate = new Date();
		let purchasedDate = dateLastPurchased
			? dateLastPurchased.toDate()
			: dateLastPurchased;
		const timeElapsed = Math.abs(currentDate - purchasedDate);
		const hoursElapsed = timeElapsed / (1000 * 60 * 60);
		hoursElapsed < 24 ? setCheck(true) : setCheck(false);`

Reply:
@sarahmjenkins Will look into! 🙂